internal/cmdrender/executor.go

```diff
@@ -253,8 +257,14 @@ func hydrate(ctx context.Context, pn *pkgNode, hctx *hydrationContext) (output [
 }
 
 // runPipeline runs the pipeline defined at current pkgNode on given input resources.
-func (pn *pkgNode) runPipeline(ctx context.Context, input []*yaml.RNode) ([]*yaml.RNode, error) {
+func (pn *pkgNode) runPipeline(ctx context.Context, hctx *hydrationContext, input []*yaml.RNode) ([]*yaml.RNode, error) {
 	const op errors.Op = "pipeline.run"
+	pr := printer.FromContextOrDie(ctx)
+	// we only print package name here
+	// TODO: the DisplayPath is a relative file path. It cannot represent the
+	// package structure. We should have function to get the relative package
+	// path here.
```

Comment:
I couldn't parse this TODO. First comment says: "we only print package name here" and since it is printing "DisplayPath" and that is relative file path not package name ?

Also, can you pl. verify the output for `kpt fn render <absolute/path/pkg/dir>` to ensure it is giving expected output ?

Reply:
Removed the first line of comment. For absolute path, the `DisplayPath` is a simple relative path.